### ISSUE TITLE
Returning Authorization Code Request Notification / Reinstall fails "Unclosed connector"

### ISSUE THREAD

**Issue by cotwild** [custom-components/alexa_media_player#836]
**Describe the bug**
After googling and asking on HA Discord channel and trying to delete/uninstall anything connected with Amazon Alexa and reinstall I don't get any further.
The Alexa integration was working fine, then it started asking for a configuration / code authorization process under the notification. I proceeded, became an email with a 6-digit code, which I entered... but then the first message appeared again. I tryed this a few times.... until I found time (and nuts) to uninstall it. So I remove the "Alexa media player" integration by deleting it under HA settings / integrations. Then I removed an existing entry in the "configuration.yaml" file. After a HA restart I filled in email, password and amazon.de, pressed OK then a failure with "Abgebrochen" (Cancelled) came up. The home assistant logfile showed:
2020-07-06 14:07:36 ERROR (MainThread) [homeassistant.core] Error doing job: Unclosed client session
2020-07-06 14:07:36 ERROR (MainThread) [homeassistant.core] Error doing job: Unclosed connector
I tried to uninstall the alexa media player integration over HACS, restarted HA and installed it again over HACS. Still no luck.
Finally full server reboot... no changes... Still error "Cancelled" with the unclosed client session / connector...

**To Reproduce**
1. Go to integrations
2. Click on '+' and search for "Alexa Media Player"
3. Fill in valid email adress, password, and "amazon.de" (advanced debug and the rest of the settings doesn't change a thing)
4. Click on "Send" and I get a popup message "Abgebrochen" (as I set German as language, "Cancelled")
5. in the homeassistant/homeassistant.log file I got the above two lines with unclosed client session / connector...

**Expected behavior**
Working Alexa Media Player integration to be able to use my Echo's again with home assistant... 

**Screenshots**

**System details**
 - Home-assistant (version): 0.112.2
 - Hassio (Yes/No): (Please note you may have to restart hassio 2-3 times to load the latest version of alexapy after an update. This looks like a HA bug). Yes
 - alexa_media  (version from `const.py` or HA startup): 2.9.2
 - alexapy (version from `pip show alexapy` or HA startup): 1.10.8

**Logs**
Please provide [logs](https://github.com/custom-components/alexa_media_player/wiki/FAQ#how-do-i-enable-debug-logging-for-the-component). We'll be most likely asking for them anyway.

2020-07-06 14:07:36 ERROR (MainThread) [homeassistant.core] Error doing job: Unclosed client session
2020-07-06 14:07:36 ERROR (MainThread) [homeassistant.core] Error doing job: Unclosed connector

**Additional context**
anything else?

**Comment by alandtse** [comment]
None of those messages are coming from this component. Please provide logs as described in the template.

**Comment by ottelo9** [comment]
I've the same problem. Delete the integration and readd it. I get the message [https://imgur.com/zD7EzfB](url). I click on the link in the email and accept the connection. But the message dont hide. Doesnt matter if I click on submit or not..

**Comment by alandtse** [comment]
@ottelo9 I'll need the logs when it's in this state to figure out what's going on. You're describing #807 which isn't what the original poster is reporting. You can either do the workaround by enabling 2FA, or if you want to help us try to fix it, please provide your logs to that issue.

**Comment by cotwild** [comment]
Sorry for the missing Log information. In the meantime I found the solution. The password I used was still correct and my stored Amazon Cookie kept me logged in on website and amazon app on mobile. But after logging out on amazon website and trying to relogin a message came, that I have to change my password. after password change and setting the new password in HA all worked flawlessly again!

Weird behaviour... but definitely not caused by your component :-) Thanks for your great work anyway!
